src/CommunityToolkit/Xamarin.CommunityToolkit/Helpers/LocalizationResourceManager.shared.cs

```diff
@@ -29,18 +29,14 @@ public void Init(ResourceManager resource, CultureInfo initialCulture)
 		public string this[string text] =>
 			GetValue(text);
 
-		public void SetCulture(CultureInfo language)
+		[Obsolete("Use " + nameof(CurrentCulture) + " to set culture")]
+		public void SetCulture(CultureInfo language) => CurrentCulture = language;
+
+		public CultureInfo CurrentCulture
 		{
-			currentCulture = language;
-			Invalidate();
+			get => currentCulture;
+			set => SetProperty(ref currentCulture, value, null);
 		}
-
-		public CultureInfo CurrentCulture => currentCulture;
-
-		public event PropertyChangedEventHandler PropertyChanged;
-
-		public void Invalidate() =>
-			PropertyChanged?.Invoke(this, new PropertyChangedEventArgs(null));
```

Comment:
@jfversluis can we just remove it? Or we should mark it as obsolete and editorbrowsablestate.never

Reply:
This method doesn't make sense anymore. Culture is stored in the private field, and when we change it, we always call `PropertyChanged` ourselves. So `public Invalidate` method will just needlessly send an event.

It was needed earlier since culture was stored in `Thread.CurrentThread.CurrentUICulture` that could be changed outside of `LocalizationResourceManager`

Reply:
I see what you're saying, I think it would make sense to first release a version where it's deprecated though. That way people will get a heads up that it is going away and gives them the opportunity to change their code at their pace instead of just dropping a breaking change over the fence that they have to deal with without any notice upfront.

Does that make sense?

Reply:
Makes sense, returned the method. When are we planning to remove everything that is marked as `Obsolete`?

Reply:
I would say 1 version obsolete, next version it's gone :)